src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/MetricItemExtensions.cs

```diff
@@ -133,7 +141,7 @@ internal static OtlpMetrics.Metric ToOtlpMetric(this IMetric metric)
                         ? OtlpMetrics.AggregationTemporality.Delta
                         : OtlpMetrics.AggregationTemporality.Cumulative,
                 };
-                var dataPoint = metric.ToNumberDataPoint(sumMetric.Sum, sumMetric.Exemplars);
+                var dataPoint = metric.ToNumberDataPoint(sumMetric.Sum.Value, sumMetric.Exemplars);
```

Comment:
Oops. Was not sending in the actual value which is expected to be a long or double.